pkgs/servers/nextcloud/packages/apps/memories-paths.diff

```diff
@@ -0,0 +1,31 @@
+diff --git a/lib/Settings/SystemConfig.php b/lib/Settings/SystemConfig.php
+index 7814f071..bb1309fa 100644
+--- a/lib/Settings/SystemConfig.php
++++ b/lib/Settings/SystemConfig.php
+@@ -124,6 +124,13 @@ class SystemConfig
+      */
+     public static function get(string $key, mixed $default = null): mixed
+     {
++        switch ($key) {
++          case "memories.exiftool": return "@exiftool@";
++          case "memories.vod.ffmpeg": return "@ffmpeg@";
++          case "memories.vod.ffprobe": return "@ffprobe@";
++          case "memories.vod.path": return "@go-vod@";
++        }
++
```

Comment:
```suggestion
@@ -124,6 +124,14 @@ class SystemConfig
      */
     public static function get(string $key, mixed $default = null): mixed
     {
+        switch ($key) {
+          case "memories.exiftool": return "@exiftool@";
+          case "memories.exiftool_no_local": return false;
+          case "memories.vod.ffmpeg": return "@ffmpeg@";
+          case "memories.vod.ffprobe": return "@ffprobe@";
+          case "memories.vod.path": return "@go-vod@";
+        }
+
```

We also need to fix `memories.exiftool_no_local` to prevent the memories from trying to use perl with the nonexistent exiftool we deleted form bin-ext